### ISSUE TITLE
Shortcut modifications propagate to home screen

### ISSUE THREAD

**Issue by zeroClearAmerican** [Waboodoo/HTTP-Shortcuts#282]
**Is your feature request related to a problem? Please describe.**
When using the shortcut modification functions in scripting, shortcuts placed on the home screen aren't picking up the modifications. For instance, if a shortcut changes the icon of itself or another shortcut, the home screen representation of that shortcut stays the same.

**Describe the solution you'd like**
When a script modifies a shortcut's title and/or icon, the modification(s) should propagate to the same home screen shortcut. 

**Describe alternatives you've considered**
I've tried restarting the device after triggering a change but this did not sync the icon or the title on the home screen with what it was in the app. 

**Additional context**
To observe this behavior, set up a POST request shortcut to "https://httpbin.org/post" with a title of "_Name_", an icon of "_flat_color_brightness_", and the below script in "**Run on success**":
```
renameShortcut('', 'Name 2');
changeIcon('', 'flat_color_brightness_3');
```
Save, place the shortcut on the home screen, then run it. The icon and title change can be seen in the app's list of shortcuts, but the home screen icon will still have the original icon and title.

**Comment by zeroClearAmerican** [comment]
My bad, I thought I submitted this as a feature request. I did not realize there was more than one placement mode. 

I've confirmed that it's using the legacy mode, without the little blue icon. How can I change the placement mode?

**Comment by Waboodoo** [comment]
I'm afraid this feature is only available on Android 7.1 and higher. On older versions only the legacy shortcuts are available, for which it is not possible to update them from the app.
